pgp/sops_functional_tests_key.asc

```diff
@@ -32,3 +32,94 @@ sZlaMHbqJX6TpOiFU8FvMU8syhBgWNYYq/3JNKw9GVDabO5Ggbluup/q1/uJHO1y
 v8bK3sq5s580DSaWamLgSLwl3UfCnL3Qa0XDT6SoY1L/Wzr1iW+9
 =zqxY
 -----END PGP PRIVATE KEY BLOCK-----
+-----BEGIN PGP PRIVATE KEY BLOCK-----
```

Comment:
I have added a new public/private key pair and a public key to this file, so that tests can use key groups.